### ISSUE TITLE
Plan abstraction of generator code into independent repositories

### ISSUE THREAD

**Issue by aeksco** [codotype/codotype-generator#51]
The following generators should be abstracted into separate repositories:
- ExpressJS
- VueJS
- ReactJS
- Flask
- Falcon

ToDos:
- Define a naming convention for all `blazeplate-generator` repositories, for example:

```
blazeplate-reactjs-simple-generator
blazeplate-expressjs-mongodb-generator
blazeplate-expressjs-postgres-generator
blazeplate-flask-postgres-generator
blazeplate-flask-mongodb-generator
blazeplate-reactjs-redux-generator
blazeplate-reactjs-reactboilerplate-generator
blazeplate-vuejs-simple-generator
blazeplate-vuejs-vuex-generator
```

- Define a pattern for each generator to expose it's primary `BlazeplateGenerator` class definition, as well as some additional metadata about the generator itself.

This task will be split up into smaller tickets that will be linked here.

**Comment by aeksco** [comment]
The Vue generator has been moved into a separate repo - over the next week I'll be making more repositories to handle this migration

**Comment by aeksco** [comment]
Resolved by #78
